invenio_i18n/cli.py

```diff
@@ -162,7 +161,41 @@ def i18n():
     help="Entrypoint group used to get package assets paths. Default: \"invenio_assets.webpack\" You don't need to set this option under normal circumstances.'",
 )
 def distribute_js_translations(input_directory: Path, entrypoint_group: str):
-    """Distribute JS/React translation files"""
+    """
+    Distribute package‑specific JavaScript translations.
```

Comment:
@utnapischtim I'm not sure if it should here but I put it here anyway. It may move to documentation in __init__.py.